js/stat.js

```diff
@@ -0,0 +1,46 @@
+window.renderStatistics = function (ctx, names, times) {
+    ctx.fillStyle = 'rgba(0, 0, 0, 0.7)';
+    ctx.fillRect(110, 20, 420, 270);
+    ctx.fillStyle = 'rgba(256, 256, 256, 1)';
+    ctx.strokeRect(100, 10, 420, 270);
+    ctx.fillRect(100, 10, 420, 270);
+    ctx.font = '16px PT Mono';
+    ctx.textBaseline = 'hanging';
+    ctx.fillStyle = 'rgba(0, 0, 0, 1)';
+    ctx.fillText('Ура вы победили!', 120, 40);
+    ctx.fillText('Список результатов:', 120, 60);
+
+    var max = -1;
+    var maxIndex = -1;
+
+    for (var i = 0 ; i < times.length; i++) {
+        var time = times[i];
+        if (time > max) {
+            max = time;
+            maxIndex = i;
+        }
+    }
+
+    var histogramHeight = -100;
+    var step = histogramHeight / (max - 0);
```

Comment:
```suggestion
    var ratio = histogramHeight / max;
```
я бы назвал `ratio` т.к. ты вычисляешь коэффициент/отношение между величинами